inspec/a2-iam-legacy-integration/files/fixtures/converge/chefdk-debian-7-tester-2d206b_run_converge.json

```diff
@@ -0,0 +1,6680 @@
+{
```

Comment:
TODO: copied these from a2-api-integration, I'd like to move them to a common location